src/api/metapointer.ts

```diff
@@ -0,0 +1,67 @@
+import * as fs from 'fs';
```

Comment:
_This comment is about the whole file._

The same with `metapointer` - why not move it to `src/metapointer.ts`?